src/modules/sensors/sensor_params.c

```diff
@@ -237,6 +237,18 @@ PARAM_DEFINE_INT32(SENS_EN_LL40LS, 0);
  */
 PARAM_DEFINE_INT32(SENS_EN_SF0X, 0);
 
+/**
+ * TFmini LiDAR
+ *
+ * @reboot_required true
+ * @min 0
+ * @max 1
+ * @group Sensors
+ * @value 0 Disabled
+ * @value 1 TFmini
```

Comment:
`@value 1 Enabled`

Reply:
Fixed

Reply:
You can use @boolean here. See SENS_EN_MB12XX for an example.